pulsar-common/src/main/java/org/apache/pulsar/common/naming/SystemTopicNames.java

```diff
@@ -81,7 +81,7 @@ public static boolean isTopicPoliciesSystemTopic(String topic) {
         if (topic == null) {
             return false;
         }
-        return TopicName.get(topic).getLocalName().equals(NAMESPACE_EVENTS_LOCAL_NAME);
+        return TopicName.get(topic).getLocalName().startsWith(NAMESPACE_EVENTS_LOCAL_NAME);
```

Comment:
to match partitioned topic name `persistent://public/default/__change_event-partition-0`

Reply:
This is strange.
it means that sometimes we call this function on the logical name of the partitioned topic and sometimes on a partition.

are we changing the behavior ?

Reply:
Ok, I can revert this change.